README.md

```diff
@@ -41,7 +41,11 @@ npm install
 
 ## Usage
 
-TBD
+See [LD Key Pair Documentation](https://github.com/digitalbazaar/crypto-ld/blob/enhancement/jsdoc-to-readme/docs/LDKeyPair.md)
```

Comment:
Is a link to the index.md file needed here?

Reply:
Not sure because index.md just contains typedefs. This is actually an unsolved problem the new linter. Custom typedefs come up as undefined so need to figure out a way of making js-doc aware of typedefs across multiple files. I will go ahead and link to index.md with a link stating Types Documentation. I might note some of those might not be necessary it is more than possible forge's own js-doc already has a type def for a few of those.

- [ ] Add Type Documentation link to index.md to README.md